src/index.ts

```diff
@@ -2,6 +2,8 @@ import { randomBytes, createHmac } from "crypto"
 
 const charset = "ABCDEFGHIJKLMNOPQRSTUVWXYZ234567"
 
+export type hashAlgorithm = "SHA1" | "SHA256" | "SHA512";
```

Comment:
Do you mind renaming this to HashAlgorithm? I believe TypeScript types follow Pascal case naming convention.

Reply:
Updated